plugin-server/src/utils/db/kafka-producer-wrapper.ts

```diff
@@ -104,8 +104,10 @@ export class KafkaProducerWrapper {
     }
 
     public async disconnect(): Promise<void> {
-        await this.flush()
-        await disconnectProducer(this.producer)
+        if (this.producer.isConnected()) {
```

Comment:
we were calling this twice in tests leading to errors... seems safe to ignore this if it's called when already disconnected